corehq/apps/geospatial/tests/test_report.py

```diff
@@ -1,13 +0,0 @@
-from django.test import TestCase
```

Comment:
I removed this test since _get_geo_location is refactored to just call `Geopoint.from_string`